service/src/main/java/uk/gov/hmcts/reform/fpl/enums/Event.java

```diff
@@ -1,53 +1,31 @@
-import static uk.gov.hmcts.reform.fpl.enums.State.RETURNED;
-import static uk.gov.hmcts.reform.fpl.enums.State.SUBMITTED;
+import lombok.RequiredArgsConstructor;
 
 @Getter
+@RequiredArgsConstructor
 public enum Event {
 
-    ORDERS_SOUGHT("ordersNeeded", "Orders and directions sought", List.of(OPEN)),
-    HEARING_URGENCY("hearingNeeded", "Hearing urgency", List.of(OPEN, RETURNED)),
-    GROUNDS("enterGrounds", "Grounds for the application", List.of(OPEN, RETURNED)),
-    RISK_AND_HARM("enterRiskHarm", "Risk and harm to children", List.of(OPEN, RETURNED)),
-    FACTORS_AFFECTING_PARENTING("enterParentingFactors", "Factors affecting parenting", List.of(OPEN, RETURNED)),
-    ////TO DO remove when toggling on FPLA-768
-    DOCUMENTS("uploadDocuments", "Upload documents",
-        List.of(OPEN, SUBMITTED, GATEKEEPING, CASE_MANAGEMENT, RETURNED)),
-    ORGANISATION_DETAILS("enterApplicant", "Applicant's details", List.of(OPEN, RETURNED)),
-    CHILDREN("enterChildren", "Child's details", List.of(OPEN, RETURNED)),
-    RESPONDENTS("enterRespondents", "Respondents' details", List.of(OPEN, RETURNED)),
-    ALLOCATION_PROPOSAL("otherProposal", "Allocation proposal", List.of(OPEN, RETURNED)),
-    OTHER_PROCEEDINGS("otherProceedings", "Other proceedings", List.of(OPEN, RETURNED)),
-    INTERNATIONAL_ELEMENT("enterInternationalElement", "International element", List.of(OPEN, RETURNED)),
-    OTHERS("enterOthers", "Other people in the case", List.of(OPEN, RETURNED)),
-    COURT_SERVICES("attendingHearing", "Court services needed", List.of(OPEN, RETURNED)),
-    SUBMIT_APPLICATION("submitApplication", "Submit application", List.of(OPEN, RETURNED)),
-    CASE_NAME("changeCaseName", "Change case name", List.of(OPEN, RETURNED)),
-    APPLICATION_DOCUMENTS("uploadDocuments", "Upload documents", List.of(OPEN, RETURNED));
+    ORDERS_SOUGHT("ordersNeeded", "Orders and directions sought"),
+    HEARING_URGENCY("hearingNeeded", "Hearing urgency"),
+    GROUNDS("enterGrounds", "Grounds for the application"),
+    RISK_AND_HARM("enterRiskHarm", "Risk and harm to children"),
+    FACTORS_AFFECTING_PARENTING("enterParentingFactors", "Factors affecting parenting"),
+    ORGANISATION_DETAILS("enterApplicant", "Applicant's details"),
+    LOCAL_AUTHORITY_DETAILS("enterLocalAuthority", "Local authority's details"),
+    CHILDREN("enterChildren", "Child's details"),
+    RESPONDENTS("enterRespondents", "Respondents' details"),
+    ALLOCATION_PROPOSAL("otherProposal", "Allocation proposal"),
+    OTHER_PROCEEDINGS("otherProceedings", "Other proceedings"),
+    INTERNATIONAL_ELEMENT("enterInternationalElement", "International element"),
+    OTHERS("enterOthers", "Other people in the case"),
+    COURT_SERVICES("attendingHearing", "Court services needed"),
+    SUBMIT_APPLICATION("submitApplication", "Submit application"),
+    CASE_NAME("changeCaseName", "Change case name"),
+    APPLICATION_DOCUMENTS("uploadDocuments", "Upload documents");
```

Comment:
Low priority suggestion: Could you alphabetise this? It would make it easier to read